content/influxdb/v2.0/tools/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Tools
+description: >
+  Tools for working with InfluxDB
```

Comment:
This could use some embellishment. What about:

> Use InfluxDB tools and other third-party integrations to interact with, manage, and visualize data in InfluxDB.